package Message; 
message UserDisplace 								//ÕËºÅ±»¶¥³ö
{ 
   optional int64     uid 		= 1;  //ÕËºÅID
} 